Source/KSCrash/Recording/Monitors/KSCrashMonitor_CPPException.cpp

```diff
@@ -184,13 +184,25 @@ catch(TYPE value)\
 #pragma mark - Public API -
 // ============================================================================
 
+static void initialize()
+{
+    static bool isInitialized = false;
+    if(!isInitialized)
+    {
+        isInitialized = true;
+        kssc_initCursor(&g_stackCursor, NULL, NULL);
```

Comment:
I think it's fine to leave this init to the very start of the app, so that you don't need to keep checking it. If cxa_throw gets intercepted, the stack cursor vtable will change.

What we can do, however, is add a warning to the empty implementation of `advanceCursor()` in KSStackCursor.c to include a warning message.